common/enableEpochs/flags.go

```diff
@@ -0,0 +1,73 @@
+package enableEpochs
+
+import "github.com/ElrondNetwork/elrond-go-core/core/atomic"
+
+type flagsHolder struct {
+	scDeployFlag                               atomic.Flag // nolint
```

Comment:
please remove the no lint and use pointers. It is safer to use that way. You will also need a constructor. 
Whoever wants to add a new flag, if he/she misses instantiating the flag it will fail starting from unit test level up until system test level.

Reply:
done, added unittest also